lib/rubyapi_rdoc_generator.rb

```diff
@@ -167,4 +168,15 @@ def format_method_source_body(method_doc)
 
     formatter.format(lexer.lex(method_src))
   end
+
+  def readwrite_string(symbol)
```

Comment:
Not sure how much it matters (especially since it's just the importer), but a frozen Hash constant would be faster than a case statement. Case statements are O(N), where a Hash lookup is O(1).

Something like:

```ruby
READWRITE_MAPPING = {
  'R' => 'read',
  'W' => 'write',
  'RW' => 'readwrite',
}.freeze

readwrite_string = READWRITE_MAPPING[attribute.rw]
```